halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/problem/ProblemSetBuilder.java

```diff
@@ -46,6 +50,6 @@ public ProblemSet build() {
         .map(ProblemBuilder::build)
         .collect(Collectors.toList());
 
-    return new ProblemSet(problems);
+    return new ProblemSet(problems).throwifSeverityExceeds(severity);
```

Comment:
saving a pair of statements here does not justify to me the non-standard change to the public interface.